examples/00-fluent/battery_pack.py

```diff
@@ -0,0 +1,454 @@
+# Copyright (C) 2021 - 2025 ANSYS, Inc. and/or its affiliates.
+# SPDX-License-Identifier: MIT
+#
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+""".. _Battery_Pack_Simulation:
+
+Simulating a 1P3S Battery Pack Using the Battery Model
+-------------------------------------------------------------
+"""
+
+# %%
+# Objective
+# ---------
+#
+# This example demonstrates a complete 1P3S battery pack simulation workflow comprising
+# one parallel string and three cells in series using the NTGK
+# (Newman-Tiedemann-Gu-Kim) electrochemical model utilizing PyFluent APIs.
+# The goal is to model and analyze battery pack behavior under a constant
+# 200 W discharge condition. This includes setting up active and passive zones,
+# defining dual-path electrical conductivity using User-Defined Scalars (UDS),
+# and applying convection cooling to simulate heat dissipation. The simulation
+# aims to monitor key performance indicators such as pack voltage,
+# maximum temperature, current flow, and heat generation over a 1500 second
+# transient run, providing insights into thermal and electrical performance
+# relevant to electric vehicle (EV) and energy storage system (ESS) design.
+
+# %%
+# Problem Description
+# -------------------
+#
+# Simulation of a 1P3S lithium-ion battery pack, consisting of three cells
+# connected in series and no parallel branches, undergoing a constant 200 W
+# discharge to represent a high-load condition typical in electric vehicle
+# operations such as acceleration or regenerative braking. Each cell has a
+# nominal capacity of 14.6 Ah, resulting in a total pack capacity of 14.6 Ah
+# and a nominal voltage range of 10.8–12.6 V, corresponding to an average cell
+# voltage of approximately 3.9 V. The discharge rate equates to about 1.17 C,
```

Comment:
Omit the space in C-rate values. Otherwise, the reader could misinterpret them. To be even clearer, we might write something like "a C-rate of 1.17C" in place of "1.17 C".